### DISCUSSION TITLE
After using the latest vscode version, the recently accessed path names on my remote server are not showing up completely

### DISCUSSION THREAD

**Discussion by ZhongLiMo** [microsoft/vscode-discussions#1325] [Extension Development QnA] [answered]
After using the latest vscode version, the recently accessed path names on my remote server are not showing up completely
Because I have several branches of code, the absolute paths are not the same
For example, there are paths such as "/data/code1/src" and "/data/code2/src", but in the new version, only "src" is displayed here, which makes me confused, I need to hover over the short path to show the full path. In previous versions, the full path is shown here

![image](https://github.com/user-attachments/assets/e658df1b-f871-424e-92da-e3a0d05620e6)

**Comment by gjsjohnmurray** [comment, marked as answer]
It sounds like you are a VS Code user rather than a developer of extensions for VS Code. Please open your issue on the main VS Code repo. I think the best way is to use the Report Issue option from the Help menu or from Command Palette. You won't be able to paste screenshots into the dialog, but after  you submit your report the GitHub page will open in your Web browser and you can edit the report or add another comment. Images can be pasted from clipboard or drag-dropped from your file Explorer. The cannot be added via email replies to GH notification emails.

**Reply by ZhongLiMo** [reply]
Yes,i'm a VS Code user.Thank you.
